contracts/CO30_Emerson_RedSkyAtNight.Tanoa/initPlayerLocal.sqf

```diff
@@ -0,0 +1,34 @@
+#include "do_not_edit\script_component.hpp"
+#include "do_not_edit\initPlayerLocal.sqf"
+
+[
+    // Add additional files here
+    // if you create "edit_me/briefing/example.html, then add "Example" here
+] call mission_fnc_briefing;
+
+// scripts
+FARE_fnc_rooftopAccess = compile preprocessFileLineNumbers "scripts\rooftopAccess.sqf";
+FARE_fnc_boatHorn = compile preprocessFileLineNumbers "scripts\boatHorn.sqf";
```

Comment:
```suggestion
```
ignored on the clients, the trigger should be
```sqf
if (isServer) then { [vehicle this] call FARE_fnc_boatHorn }:
```
to just avoid this on the client altogether